samples/generated/static-spa/public/app.js

```diff
@@ -335,6 +354,7 @@ function showForm() {
   // Set values from config
   document.getElementById('issuer').value = config.issuer;
   document.getElementById('clientId').value = config.clientId;
+  document.getElementById('scopes').value = config.scopes;
```

Comment:
I think it would be less confusing if `config.scopes` in the samples was an array, consistent with what okta-auth-js expects. The conversion to/from string can be handled here in `showForm`. In the default test app, the array is converted to a comma separated string: https://github.com/okta/okta-auth-js/blob/master/test/app/src/form.ts#L51
Within the JSON serialized state param, it can remain an array. It only needs to be a string for the 2 cases:
- in the form
- as a standalone query parameter
These 2 cases are closely related because the form has method GET so submitting the form will set the query parameter with the value in the form.  The serialized state object is used to save the config across redirects.